pkg/multicluster/virtual_cluster.go

```diff
@@ -49,6 +45,18 @@ type VirtualCluster struct {
 	Metrics  *ClusterMetrics
 }
 
+// NewVirtualClusterFromLocal return virtual cluster corresponding to local cluster
+func NewVirtualClusterFromLocal() *VirtualCluster {
+	return &VirtualCluster{
+		Name:     ClusterLocalName,
+		Type:     types.CredentialTypeInternal,
+		EndPoint: "-",
```

Comment:
nit: explicitly clarify the meaning of the "-" here, or define it as a named variable